crates/ruff_python_parser/src/parser/statement.rs

```diff
@@ -1466,19 +1469,15 @@ impl<'src> Parser<'src> {
         // # parse_options: {"target-version": "3.10"}
         // try: ...
         // except* ValueError: ...
-
-        let range = self.node_range(try_start);
-        if is_star {
-            self.add_unsupported_syntax_error(UnsupportedSyntaxErrorKind::ExceptStar, range);
-        }
+        // except* KeyError: ...
```

Comment:
We can add some whitespaces around the star operator like `except  *    KeyError` to make sure that the highlighted range is still correct.